.eslintrc.json

```diff
@@ -1,22 +1,23 @@
 {
   "root": true,
-  "plugins": ["prettier"],
-  "rules": {
-    "prettier/prettier": "warn"
-  },
-  "extends": ["eslint:recommended", "plugin:prettier/recommended"],
-  "env": {
-    "browser": true,
-    "jasmine": true
-  },
   "overrides": [
     {
       "files": ["*.ts"],
+      "parser": "@typescript-eslint/parser",
       "parserOptions": {
         "project": ["tsconfig.*?.json"],
         "createDefaultProgram": true
       },
-      "extends": ["plugin:@typescript-eslint/recommended", "plugin:@angular-eslint/recommended"],
+      "plugins": ["@typescript-eslint"],
+      "extends": [
+        "plugin:@angular-eslint/recommended",
+        "plugin:@typescript-eslint/recommended",
```

Comment:
This includes rules from `eslint:recommended` (see [note](https://typescript-eslint.io/docs/linting/configs/#recommended)) hence we can remove it.